yt_dlp/extractor/afl.py

```diff
@@ -0,0 +1,204 @@
+from .brightcove import BrightcoveNewIE
+from .common import InfoExtractor
+from .omnyfm import OmnyFMShowIE
+from ..utils import (
+    extract_attributes,
+    get_element_by_class,
+    get_element_html_by_attribute,
+    get_element_html_by_id,
+    smuggle_url,
+    str_or_none,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class AFLVideoIE(InfoExtractor):
+    IE_NAME = 'afl:video'
+    _VALID_URL = r'https?://(?:www\.)?(?:afl|lions)\.com.au/(?:aflw/)?video/(?P<id>\d+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?(?:afl|lions)\.com\.au/(?:aflw/)?video/(?P<id>\d+)'
```

Reply:
Wondering how the earlier regex even worked in the first place. I've added the fix.